README.md

```diff
@@ -1,94 +1,127 @@
+| snapchat__ad_report          | Each record represents the daily ad performance of each ad.                                                            |
+| snapchat__UTM_report         | Each record represents the daily ad performance of each UTM parameter.                                                           |
+| snapchat__ad_squad_report    | Each record represents the daily ad performance of each ad squad.         
+# 🎯 How do I use the dbt package?
 
-## Installation Instructions
+## Step 1: Prerequisites
+To use this dbt package, you must have the following:
 
-Check [dbt Hub](https://hub.getdbt.com/) for the latest installation instructions, or [read the dbt docs](https://docs.getdbt.com/docs/package-management) for more information on installing packages.
+- At least one Fivetran Snapchat Ads connector syncing data into your destination.
+- A **BigQuery**, **Snowflake**, **Redshift**, **Databricks**, or **PostgreSQL** destination.
 
-Include in your `packages.yml`
+### Databricks Dispatch Configuration
+If you are using a Databricks destination with this package you will need to add the below (or a variation of the below) dispatch configuration within your `dbt_project.yml`. This is required in order for the package to accurately search for macros within the `dbt-labs/spark_utils` then the `dbt-labs/dbt_utils` packages respectively.
+```yml
+dispatch:
+  - macro_namespace: dbt_utils
+    search_order: ['spark_utils', 'dbt_utils']
+```
 
+## Step 2: Install the package
+Include the following snapchat_ads_source package version in your `packages.yml` file:
+> TIP: Check [dbt Hub](https://hub.getdbt.com/) for the latest installation instructions or [read the dbt docs](https://docs.getdbt.com/docs/package-management) for more information on installing packages.
 ```yaml
 packages:
   - package: fivetran/snapchat_ads
-    version: [">=0.3.0", "<0.4.0"]
-```
+    version: [">=0.4.0", "<0.5.0"]
 
-## Configuration
+```
+## Step 3: Configure your variables
 
-By default, this package will look for your Snapchat Ads data in the `snapchat_ads` schema of your [target database](https://docs.getdbt.com/docs/running-a-dbt-project/using-the-command-line-interface/configure-your-profile). If this is not where your Snapchat Ads data is, please add the following configuration to your 
-`dbt_project.yml` file:
+### Define database and schema variables
+By default, this package runs using your destination and the `snapchat_ads` schema. If this is not where your Snapchat Ads data is (for example, if your Snapchat Ads schema is named `snapchat_ads_fivetran`), add the following configuration to your root `dbt_project.yml` file:
 
 ```yml
-# dbt_project.yml
-
-...
-config-version: 2
-
 vars:
-    snapchat_schema: your_schema_name
-    snapchat_database: your_database_name 
+    snapchat_ads_database: your_destination_name
+    snapchat_ads_schema: your_schema_name 
```

Comment:
These variables in the source are actually `snapchat_schema` and `snapchat_database`